lib/server/controllers/logs.controller.js

```diff
@@ -281,6 +286,40 @@ const getLogById = async (request, response, next) => {
     }
 };
 
+/**
+ * Get all root logs
+ *
+ * @param {Object} request The *request* object represents the HTTP request and has properties for the request query
+ *                         string, parameters, body, HTTP headers, and so on.
+ * @param {Object} response The *response* object represents the HTTP response that an Express app sends when it gets an
+ *                          HTTP request.
+ * @returns {undefined}
+ */
+const getRootLogsHandler = async (request, response) => {
+    const value = await dtoValidator(
```

Comment:
Your variable names should always mean things. For example here, I would suggest `requestDto` instead of value.

Reply:
done